src/wallet/scriptpubkeyman.h

```diff
@@ -292,8 +292,7 @@ class LegacyScriptPubKeyMan : public ScriptPubKeyMan, public FillableSigningProv
 
     void MarkUnusedAddresses(const CScript& script) override;
 
-    //! Upgrade stored CKeyMetadata objects to store key origin info as KeyOriginInfo
```

Comment:
In commit "Locking: Lock cs_KeyStore instead of cs_main in legacy keyman" (90287119f4bab55e3ffb8e5b9139d19f11eebd61):

Looks like comment this line was unintentionally deleted (mistake during rebase)

Reply:
Added it back in.